cmake/Re2cTryCXXFlag.cmake

```diff
@@ -0,0 +1,16 @@
+include(CheckCXXCompilerFlag)
+# Iff C++ compiler recognizes 'flag', append 'flag' and 'implied-flags' to CXXFLAGSDEFAULT
```

Comment:
Is 'Iff'  here means 'if and only if'? Don't you think this is bit nerdy? :)

Reply:
@sergeyklay It comes from the configure.ac, which was added by me back in the days when I was an undergrad student and thought the world could work on 1st order predicate logic. :D

@ligfx just did a good job at porting the comment as well.

Reply:
Actually, that was not so long ago: https://github.com/skvadrik/re2c/commit/4c7a3c83c18e13455d0f30b1c445e7831f0c85b7

Reply:
@skvadrik @sergeyklay I mean it's not wrong, and you could always go more nerdy and start using the symbols like ⇔ !

Reply:
Not until re2c is rewriten in haskell. ;)